atomate/vasp/firetasks/glue_tasks.py

```diff
@@ -292,3 +295,77 @@ def run_task(self, fw_spec):
                                "eigenvecs": normalmode_eigenvecs.tolist(),
                                "norms": normalmode_norms.tolist()}
         return FWAction(mod_spec=[{'_set': {'normalmodes': normalmode_dict}}])
+
+
+@explicit_serialize
+class TransferNEBTask(FiretaskBase):
```

Comment:
I don't think this needs to be in glue_tasks. All NEB firetasks can be consolidated under a NEB subpackage for now. Eventually, a TransferDirectory task should be written that does a full transfer of a directory, rather than having separate Transfer* for different types of run.